starter_code/app.js

```diff
@@ -16,6 +20,27 @@ app.get('/', (req, res, next) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res, next) => {
+   punkAPI.getBeers()
+  .then(beers => {
+    res.render('beers', {beers});
+  })
+  .catch(error => {
+    console.log(error)
+  })
+});
+
+app.get('/random-beer', (req, res, next) => {
+  punkAPI.getRandom()
+  .then(beers => {
+    let beer = _.sample(beers);
+    res.render('randomBeer', {beer});
+  })
+  .catch(error => {
+    console.log(error)
+  })
+})
+
 
 
 app.listen(3000);
```

Comment:
Use a callback inside the app.listen with a console log in order to let you know when a server is running 😉 
`const PORT = 3000`

``` 
app.listen(PORT, ()=>{
  console.log(`listen on port: ${PORT}`)
});